wiki/SourceCode.md

```diff
@@ -82,8 +82,8 @@ Biopython developers (including all those who previously had CVS commit
 rights).
 
 This is normally given on a case by case basis, and the best place to
-discuss getting write access is on the [Biopython Development mailing
-list](mailto:biopython-dev@biopython.org).
+discuss getting write access is on the [Biopython mailing
+list](mailto:biopython@biopython.org).
```

Comment:
Maybe it would be better to link to the wiki page instead?

i.e. https://biopython.org/wiki/Mailing_lists

What do you think?